src/utilities/auth.js

```diff
@@ -0,0 +1,18 @@
+const setToken = (token) => {
+  if (localStorage.getItem('token')) {
```

Comment:
so if a token exists we remove it and then set a new token? Why?

Reply:
I was thinking of a situation where a person had to re-login because of maybe an expired token. I just clarified that there's no need to clear it first. It can just be overridden.